microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -2169,7 +2169,8 @@ Not applicable.
 Manage the user data folder (UDF), which is a folder on the user's machine.  The UDF contains data related to the host app and WebView2.  WebView2 apps use user data folders to store browser data, such as cookies, permissions, and cached resources.
 
 See also:
-* [Manage user data folders](user-data-folder.md)
+* [Manage user data folders](./user-data-folder.md)
+* [Clear browsing data from the user data folder](./clear-browsing-data.md)
```

Comment:
May I ask why is this change in this PR?

Reply:
@victorhuangwq 
Adding the period is trivial and not the focus - this is standard pattern per the tools, standard for this repo.

Adding the "Clear browsing data" link is tying up loose ends from previous release notes & subsequently updating the "Overview of APIs" article to add Stable Release APIs.  This change patches an omission from the previous "Overview of APIs" PR.  The nearby tab-set added a "clear data" API, and therefore the section needed a link to the article about that API.

Note that the present Release RelNotes PR won't need such a follow-on PR to add items to "Overview of APIs" article, since this release doesn't add any Stable Release APIs.